crowdsourcing/models.py

```diff
@@ -1,13 +1,15 @@
 from django.contrib.auth.models import User
 from django.db import models
+from model_utils import Choices
```

Comment:
true; we always have to check what makes sense to write by ourselves and what not, we have to see what functionality they bring to us, for this particular case since these statuses are being used across the app we create a python module `status.py` and import it as `from crowdsourcing import status` and use it in Models, Serializers, Viewsets wherever; the constants can follow a certain pattern  MODELNAME_STATUS -> e.g PROJECT_PUBLISHED